tools/alpine/Makefile

```diff
@@ -2,24 +2,36 @@
 
 ORG?=linuxkit
 IMAGE=alpine
+
+DOCKER_CONTENT_VAR=1
+PKG_DEPS=packages
+ifeq ($(shell uname -m), x86_64)
 BASE=alpine:3.6
+PKG_DEPS += packages.x86_64
+endif
+
+ifeq ($(shell uname -m), aarch64)
+BASE=arm64v8/alpine:3.6
+PKG_DEPS += packages.aarch64
+DOCKER_CONTENT_VAR=0
+endif
```

Comment:
The reason is, if we export DOCKER_CONTENT_TRUST=1 for x86, then we will encounter the issue filed by https://github.com/moby/moby/issues/34199 no matter we use:
$docker build --no-cache --build-arg BASE=alpine:3.6 -t alpine:build .
or
$docker build --no-cache  -t alpine:build .

That's why we still use the original Makefile content trust mode when building the image.